app-admin/mkosi/mkosi-v11.ebuild

```
@@ -0,0 +1,40 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="A tool for generating OS images"
+HOMEPAGE="https://github.com/systemd/mkosi"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="+debian"
+KEYWORDS=""
+RDEPEND="
+	sys-apps/systemd
+	dev-vcs/git
+	debian? ( dev-util/debootstrap )
+"
+
+distutils_enable_tests pytest
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="${HOMEPAGE}.git"
+	EGIT_COMMIT="v${PVR}"
+	EGIT_COMMIT="v${PVR}"
+	if [[ "${PV}" == "9999" ]]; then
+		MY_KEYWORDS=""
+	fi
+else
+	SRC_URI="https://github.com/systemd/mkosi/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
```

Missing quote.
```suggestion
	SRC_URI="https://github.com/systemd/mkosi/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
```

Make sure to test ;)